docs/deployment/index.rst

```diff
@@ -22,6 +23,8 @@ Web Servers
    gunicorn
    nginx
    uwsgi
+   uwsgi_cookiecutter_1__nginx
+   uwsgi_cookiecutter_2__emperor
```

Comment:
Please condense double-underscores to singles, and rename the files accordingly.